src/apps/vpn/ui/screens/getHelp/developerMenu/ViewDeveloperMenu.qml

```diff
@@ -64,18 +64,20 @@ MZViewBase {
                 if (MZSettings.stagingServerAddress !== serverAddressInput.text) {
                     MZSettings.stagingServerAddress = serverAddressInput.text;
                 }
-            }
-
-            Component.onCompleted: {
-                serverAddressInput.text = MZSettings.stagingServerAddress;
-            }
 
-            Component.onDestruction: {
-                // If a server address is not specified, don't use a staging server
+                // If the server address has been deleted, uncheck checkBoxRowStagingServer to not use a
+                // staging server. Also reset the server address to the default, so the user can easily switch
+                // back to it by rechecking the checkbox.
                 if (serverAddressInput.text.length == 0) {
                     MZSettings.stagingServer = false;
```

Comment:
Again, since this is just the dev menu I think this is fine - but I'm not a huge fan of the pattern of disabling the text field right when the text length is 0. It's rare, but if I was looking to change this url, I would most likely start by double-clicking the current url to highlight it and hit backspace, which, in this case, would result in the field being disabled/reset.

Any issue with just doing this logic on destruction?

Reply:
Ahhh, I see where this would cause a crash when quitting with a blank staging URL. Is there any opportunity to mitigate within the C++ code on startup to revert an empty URL to the default staging address?

Reply:
>  I would most likely start by double-clicking the current url to highlight it and hit backspace, which, in this case, would result in the field being disabled/reset.

Good observation!

Reply:
The fix has been changed to handle this. When staging is set on app startup, an empty address is changed to the default address.